lib/LocalizationProvider.js

```diff
@@ -91,6 +92,13 @@ function LocalizationProvider($localizationLoader, l10n) {
 LocalizationProvider.prototype._cookieConfig = null;
 
 /**
+ * Return localization key in case of absent value.
```

Comment:
I think it's better "Current state of returning placeholder if a value is not found.". Now it seems confusing like this is a method.